presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -739,6 +740,15 @@ public void testMultiIndexAlias()
                 "SELECT (SELECT count(*) FROM region) + (SELECT count(*) FROM nation)");
     }
 
+    @Test
+    public void testCount()
+    {
+        assertQuery("SELECT count(*) FROM orders");
+        assertQuery("SELECT count(*) FROM orders WHERE orderkey > 10");
+        assertQuery("SELECT count(*) FROM (SELECT * FROM orders LIMIT 10)");
+        assertQuery("SELECT count(*) FROM (SELECT * FROM orders WHERE orderkey > 10 LIMIT 10)");
```

Comment:
as we are heading towards more agg pushdowns, you can add this as generic test cases in 
`io.prestosql.testing.AbstractTestIntegrationSmokeTest#testCountAll`